br/pkg/task/backup.go

```diff
@@ -77,6 +78,13 @@ type BackupConfig struct {
 	IgnoreStats      bool          `json:"ignore-stats" toml:"ignore-stats"`
 	UseBackupMetaV2  bool          `json:"use-backupmeta-v2"`
 	CompressionConfig
+
+	// for ebs-based backup
+	FullBackupType      FullBackupType `json:"full-br-type" toml:"full-br-type"`
```

Comment:
```suggestion
	FullBackupType      FullBackupType `json:"full-backup-type" toml:"full-backup-type"`
```
should we change this?